README.md

```diff
@@ -120,6 +120,23 @@ helm install "${SKYWALKING_RELEASE_NAME}" ${REPO}/skywalking -n "${SKYWALKING_RE
   --set ui.image.tag=9.2.0
 ```
 
+Because ElasticSearch recommends to use the corresponding Helm Chart version of the ElasticSearch version,
+if you want to use a specific version of ElasticSearch, you have to change the ElasticSearch Helm Chart version in
+`dependencies` section in `Chart.yaml` file, which requires you to install from the source codes.
```

Comment:
Install from source codes? Meaning?

Reply:
> Install from source codes? Meaning?

Clone this repo and install, comparing to install using Docker Hub Helm Chart repository